### PR TITLE
Dockerization

### PR DESCRIPTION
**Pull request by Puckoland**

### CODE REVIEW

**Comment by Puckoland** [comment]
resolves #3

File: server/kotc_database/src/lib.rs [hunk 1]
```diff
@@ -33,9 +31,7 @@ pub async unsafe fn establish_connection() {
     match VAL {
         Some(_) => (),
         None => {
-            dotenv().ok();
-
-            let database_url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
+            let database_url = "postgres://postgres:password@db/kotc".to_string();
```
Comment (danieltimko) [inline server/kotc_database/src/lib.rs:34]: eliminate hardcoded credentials

Comment (danieltimko) [inline server/kotc_database/src/lib.rs]: eliminate hardcoded credentials

**Review by danieltimko — CHANGES_REQUESTED** [review]

**Review by revolko — APPROVED** [review]

**Review by danieltimko — APPROVED** [review]

File: server/kotc_database/src/lib.rs [hunk 2]
```diff
@@ -33,7 +32,10 @@ pub async unsafe fn establish_connection() {
     match VAL {
         Some(_) => (),
         None => {
-            let database_url = "postgres://postgres:password@db/kotc".to_string();
+            let database_url = match option_env!("DATABASE_URL") {
+                Some(url) => url.to_string(),
+                _ => "postgres://postgres:password@db/kotc".to_string(),
```
Comment (danieltimko) [inline server/kotc_database/src/lib.rs:37]: Perhaps it would be better to panic when env var is not set? Easier to debug what's going on

**Comment by Puckoland** [comment]
resolves #68

**Pull request merged**